svelte/components/Global/Home.svelte

```diff
@@ -36,7 +36,7 @@
     <Title />
 
     {#if account && getCreate(chainId)}
-      <Create {chainId} />
+      <Create {chainId} bind:refresh />
```

Comment:
warning: refresh and refreshing are bind multiple times, use another name or another method ?